validphys2/src/validphys/core.py

```diff
@@ -251,9 +264,13 @@ def __str__(self):
     def __iter__(self):
         return iter((self.datafile, self.sysfile, self.plotfiles))
 
+    # TODO: one of the two functions below needs to go
```

Comment:
I agree with this, and I would tend to say that we can just keep `load`.

However, do we want to change this in this PR, or in a later one?

Note that there are some dependencies on `load_commondata_instance` in several modules, e.g. `pseudodata.py`

Reply:
At the end of the PR, just before merging, I can do a search or replace of `load_commondata_instance` by `load`.